bmwqemu.pm

```diff
@@ -975,6 +977,16 @@ sub _scale_timeout {
     return $timeout * ($vars{TIMEOUT_SCALE} // 1);
 }
 
+# just a random string useful for pseudo security or temporary files
+sub random_string {
+    my ($count) = @_;
+    $count //= 4;
```

Comment:
yeah, I was thinking more in line random_string($value_from_outer_space). to ensure we will not compromise something by reusing some other var. But I don't oppose strongly. :)